e2e-tests/shared-test-assertions/iban-field-validation/index.js

```diff
@@ -0,0 +1,45 @@
+import { field as fieldSelector } from '../../pages/shared';
+import { POLICY as POLICY_FIELD_IDS } from '../../constants/field-ids/insurance/policy';
+import { MINIMUM_CHARACTERS, MAXIMUM_CHARACTERS } from '../../constants';
+import { ERROR_MESSAGES } from '../../content-strings';
+
+const {
+  LOSS_PAYEE_FINANCIAL_INTERNATIONAL: {
+    IBAN: FIELD_ID,
+  },
+} = POLICY_FIELD_IDS;
+
+const errorMessages = ERROR_MESSAGES.INSURANCE.POLICY[FIELD_ID];
+
+/**
+ * ibanFieldValidation
+ * Assert IBAN validation
+ * @returns {Function} Mocha describe block with assertions.
```

Comment:
do we need the returns comment?

Reply:
Yes IMO 😄  These test assertion functions don't specifically return something (with `return ...`), the assertions are returned or rather executed, without the need for a manual `return`